sycl/source/detail/queue_impl.hpp

```diff
@@ -400,10 +400,12 @@ class queue_impl {
 
   void initHostTaskAndEventCallbackThreadPool();
 
-  /// Stores a USM operation event that should be associated with the queue
+  /// queue_impl.addSharedEvent tracks events with weak pointers
```

Comment:
I assume `addSharedEvent` should have been something else on this line. Feel free to address this minor comment separately.

Reply:
thanks